internal/routing/reader.go

```diff
@@ -9,6 +9,12 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+type LocalNetwork struct {
+	Subnet        net.IPNet
```

Comment:
Maybe change that to `IPNet` instead of `Subnet` as it's not necessarily a subnet 🤔 

Also having it as `*net.IPNet` might be more appropriate to fit how golang uses it in its standard lib (always as a pointer).

Reply:
Ah no worry, I'll merge as is 😉  Thanks!